modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -26,6 +26,12 @@ const INITIAL_EDITING_LOCALE = {
 	localeId: themeDisplay.getDefaultLanguageId(),
 };
 
+const ALERT_REGEX = /alert\((.*?)\)/;
+const INNER_HTML_REGEX = /innerHTML\s*=\s*.*?/;
+const ON_ERROR_REGEX = /onerror\s*=\s*.*?/;
+const ON_LOAD_REGEX = /onload\s*=\s*.*?/;
+const ON_MOUSE_OVER_REGEX = /onmouseover\s*=\s*.*?/;
+
```

Comment:
Hey @danvb112, As mentioned by you, let's check if using a library like Dom Purify can make things easier here. let me know when we get a response from FE-Infra